entrypoint.sh

```diff
@@ -38,30 +39,36 @@ if [ -z "${ACTION}" ]; then
 fi
 
 case "${ACTION}" in
-  $SYNC_CMD)
+  "$SYNC_CMD")
     verifyTenantAndAddress
-    OUTPUT=$(/usr/bin/cortextool rules sync --rule-dirs="${RULES_DIR}" ${NAMESPACES:+ --namespaces=${NAMESPACES}} "$@")
+    OUTPUT=$(/usr/bin/cortextool rules sync --rule-dirs="${RULES_DIR}" ${NAMESPACES:+ --namespaces=${NAMESPACES}} "$@" | \
+        awk 'BEGIN {RS=""}{gsub(/\r/, "%0D") gsub(/\n/, "%0A") gsub(/%/,"%25")}1')
```

Comment:
I wonder whether this breaks `STATUS=$@`. I don't know how to run it on `$OUTPUT`.

Reply:
What about 

```
SINGLE_LINE_OUTPUT=$(echo "${OUTPUT}" | awk 'BEGIN {RS=""}{gsub(/\r/, "%0D") gsub(/\n/, "%0A") gsub(/%/,"%25")}1')
```

?

Also, not sure if the order of the `gsub`s matters here.

Reply:
This is much better. An yes. I think the order matters. I've changed it.